docs/howtos/adding-a-new-component.md

```diff
@@ -162,8 +159,6 @@ The result should look something like this:
 
 Click on the top-level "MozillaTestServices" project in the navigator, then go to "Build Phases".
 
-> Double-check that `<your_crate_name>.udl` does **not** appear in the "Copy Bundle Resources" section.
-
 Add `<your_crate_name>.udl` to the list of "Compile Sources". This will trigger an Xcode Build Rule that generates
```

Comment:
I tried to update this as best I could, but I was unsure of some Swift things.
- Do my current changes make sense?
- What should we do with the above line?  In a proc-macro world, we won't have a UDL file to add to compile sources.  Do we need to add some other file?  Or is this not needed anymore?

Reply:
I think your current changes make sense!

For the above line...I'm not sure just yet! How do we invoke `uniffi-bindgen` to generate the Swift bindings, and what does it take for the input, in the proc-macro world?

The Build Rule that we have now takes all `.*udl` files as inputs, and runs `uniffi-bindgen generate` on each one; but we can change it to whatever it needs!

Reply:
The new command is `cargo uniffi-bindgen generate --library [path-to-megazord-ios-a] -l swift -o [out-dir]`.  The input is the megazord library, which is awkward to specify in the compile sources.  Can we add a build step that unconditionally runs that after the megazord is built?  Or is my change to `build-xcframework.sh` already enough?